src/include/network/network_state.h

```diff
@@ -40,5 +37,17 @@ enum class WriteState {
   WRITE_ERROR,      // Some error happened
 };
 
+enum class Transition {
+  NONE,
+  WAKEUP,
+  PROCEED,
+  NEED_DATA,
+  // TODO(tianyu) generalize this symbol, this is currently only used in process
+  GET_RESULT,
```

Comment:
Since we have a `CONN_GET_RESULT` state, it may be better to change the `GET_RESULT` transition to some other name.

Reply:
The intention is for this to have the same name since this is the only state that makes use of this transition symbol.